sdk/storage/azure-storage-blob/tests/_shared/settings_real.py

```diff
@@ -0,0 +1,17 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# ----------------------------------------------------------------s----------
+
+STORAGE_ACCOUNT_NAME = "emilydevtest"
+STORAGE_ACCOUNT_KEY = ""
```

Comment:
replace this with your storage account name/key (I can also send you the key to emilydevtest offline)